packages/web/src/components/organisms/PropertyHeader/index.tsx

```diff
@@ -173,7 +173,7 @@ export const PropertyHeader = ({ propertyAddress, apy, creators }: Props) => {
             ? `${data?.property_authentication?.[0]?.authentication_id}'s Pool`
             : `${propertyAddress} Pool`}
         </Header>
-        {isSelf && <CopyBadge propertyAddress={propertyAddress} />}
+        {!isNotSelf && <CopyBadge propertyAddress={propertyAddress} />}
```

Comment:
This change does not seem to have solved the problem.
The following will not be displayed until the data fetch is completed, and can be determined by the results of the completed data fetch.

```suggestion
        {dataAuthor && !isNotSelf && <CopyBadge propertyAddress={propertyAddress} />}
```